crates/oxc_linter/src/rules/react/jsx_props_no_spread_multi.rs

```diff
@@ -0,0 +1,100 @@
+use std::collections::HashMap;
+
+use oxc_ast::{ast::JSXAttributeItem, AstKind};
+use oxc_diagnostics::OxcDiagnostic;
+use oxc_macros::declare_oxc_lint;
+use oxc_span::{Atom, Span};
+
+use crate::{context::LintContext, rule::Rule, AstNode};
+
+fn jsx_props_no_spread_multi_diagnostic(span0: Span, span1: Span) -> OxcDiagnostic {
+    OxcDiagnostic::warn("Disallow JSX prop spreading the same identifier multiple times.")
+        .with_help("Remove duplicate spread attributes.")
+        .with_labels([span0, span1])
```

Comment:
```suggestion
        .with_labels([first_spread.label("It is first spread here"), second_spread.label("and spread again here")])
```

Reply:
The span now accepts a single Vec due to the following change:
https://github.com/oxc-project/oxc/pull/4866/commits/7e73647ad9f4d4bf84102c400552e9521c20a290#diff-53bed08b3e6dd9253602fb036bf00377457270aaf435ab91b2f4fcbe49da6be8R10

Therefore, I’m currently passing `spans`. Is any further correction needed?
https://github.com/oxc-project/oxc/pull/4866/commits/7e73647ad9f4d4bf84102c400552e9521c20a290#diff-53bed08b3e6dd9253602fb036bf00377457270aaf435ab91b2f4fcbe49da6be8R13